File: objectbox-gradle-plugin/test-gradle-projects/java-android/src/test/java/io/objectbox/test/TestProjectTest.java
package io.objectbox.test;

import org.junit.Test;


import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotNull;
import static org.junit.Assert.assertTrue;

/** Tests some basic functionality without touching BoxStore (which would require native libs). */
public class TestProjectTest {

    @Test
    public void testBasics() {
        assertNotNull(MyObjectBox.builder());
        assertTrue(Customer_.__ALL_PROPERTIES.length >= 2);
        assertEquals(1, Order_.__ID_PROPERTY.id);
    }

}

File: objectbox-gradle-plugin/test-gradle-projects/kotlin-java-entities-android/src/test/kotlin/io/objectbox/test/kotlin/TestProjectTest.kt
package io.objectbox.test.kotlin

import io.objectbox.BoxStoreBuilder
import io.objectbox.EntityInfo

import org.junit.Test

import org.junit.Assert.assertEquals
import org.junit.Assert.assertNotNull
import org.junit.Assert.assertNull
import org.junit.Assert.assertTrue

/** Tests some basic functionality without touching BoxStore (which would require native libs).  */
class TestProjectTest {

    @Test
    fun testBasics() {
        assertNotNull(MyObjectBox.builder())
        assertTrue(Customer_().allProperties.size >= 2)
        assertEquals(1, Order_().idProperty.id)
    }


}
